src/test/java/org/mariadb/jdbc/BaseTest.java

```diff
@@ -27,7 +27,7 @@
  */
 @Ignore
 public class BaseTest {
-    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root";
+    protected static final String mDefUrl = "jdbc:mysql://localhost:3306/testj?user=root&password=root";
```

Comment:
please remove this, travis test consider having no password. 
for your tests, you can add -DdbURL=jdbc:mysql://localhost:3306/testj?user=root&password=root
that will replace this default string